torch/package/package_exporter.py

```diff
@@ -39,7 +39,6 @@
 
 ActionHook = Callable[["PackageExporter", str], None]
 
-
```

Comment:
nit: try not to include unrelated changes like whitespace and formatting in your PR to keep the project history clean.

Reply:
ah gotcha, so just lint right before merging then